frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -346,6 +348,10 @@ export const experimentLogic = kea<experimentLogicType>([
             actions.updateExperiment({ archived: true })
             values.experiment && actions.reportExperimentArchived(values.experiment)
         },
+        restartExperiment: async () => {
+            actions.updateExperiment({ start_date: null, end_date: null })
```

Comment:
Any specific reason it's not this?

```suggestion
            actions.updateExperiment({ start_date: undefined, end_date: undefined })
```

Reply:
Cool